src/node.h

```diff
@@ -42,6 +42,12 @@
 
 #define NODE_DEPRECATED(msg, fn) V8_DEPRECATED(msg, fn)
 
+#include "imembase.h"
+
+#define malloc(size) ikmem_malloc((size_t)size)
+#define realloc(ptr, size) ikmem_realloc((void *)ptr, (size_t)size)
+#define free(ptr) ikmem_free((void *)ptr)
```

Comment:
@bnoordhuis have all memory allocations been standardized to use `*alloc()`/`free()`? IIRC they don't, and there is at least one location where `delete` may be called on memory that's been `malloc()`'ed. This isn't safe to just do everywhere.